src/alfasim_sdk/_internal/cli.py

```diff
@@ -104,6 +109,11 @@ def new(dst, caption, plugin_id, author_name, author_email):
         "}",
     ]
 
+    requirements = {
+        "sdk_version": get_extras_default_required_version(),
```

Comment:
Let's rename `get_extras_default_required_version` to better reflect what it does, and for consistency with `get_required_python_version`:

```suggestion
        "sdk_version": get_required_sdk_version(),
```